### PR TITLE
feat(s3): add api pages MTA-1451

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

File: storage/object/api-cli/installing-rclone.mdx [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+dates:
+  validation: 2021-08-30
+  posted: 2021-08-25
+---
+
+[Rclone](https://rclone.org) is a command-line tool that can be used to manage your cloud storage. It communicates with any S3 compatible cloud storage provider as well as other storage platforms.
+
+Follow the instructions given in the [official rclone documentation here](https://rclone.org/install/) to install rclone. 
+
+    For example, on Linux:
```
Comment (angristan) [inline storage/object/api-cli/installing-rclone.mdx:17]:
```suggestion
For example, on Linux:
```

File: storage/object/api-cli/installing-rclone.mdx [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+
+1. Connect to your server as `root` via [SSH](/compute/instances/how-to/connect-to-instance/).
+
+2. Update the APT packet cache and the software already installed on the instance:
+  ```
+  apt update && apt upgrade -y
+  ```
+
```
Comment (angristan) [inline storage/object/api-cli/installing-rclone.mdx:25]: I think we can remove this step. It is not relevant to the tutorial, as `rclone` is installed as a standalone binary from a direct URL.

File: storage/object/api-cli/installing-rclone.mdx [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+    \ "webdav"
+  35 / Yandex Disk
+    \ "yandex"
+  36 / http Connection
+    \ "http"
+  37 / premiumize.me
+    \ "premiumizeme"
+  38 / seafile
+    \ "seafile"
+  Storage>
```
Comment (angristan) [inline storage/object/api-cli/installing-rclone.mdx:141]: Could this be truncated? It takes a lot of space

```suggestion
  Choose a number from below, or type in your own value
  1 / 1Fichier
    \ "fichier"
  2 / Alias for an existing remote
    \ "alias"
  3 / Amazon Drive
    \ "amazon cloud drive"
  4 / Amazon S3 Compliant Storage Provider (AWS, Alibaba, Ceph, Digital Ocean, Dreamhost, IBM COS, Minio, Tencent COS, etc)
    \ "s3"

[...]

  Storage>
```

**Review by angristan — APPROVED** [review]

File: storage/object/api-cli/installing-rclone.mdx [hunk 4]
```diff
@@ -0,0 +1,271 @@
+---
```
Comment (RoRoJ) [inline storage/object/api-cli/installing-rclone.mdx:1]: this whole page is giving me a 404 when I try to view with a yarn develop

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**